src/FileDownloader.ts

```diff
@@ -123,7 +123,7 @@ export default class FileDownloader implements IFileDownloader {
                 return Uri.file(fileDownloadPath);
             };
 
-            return RetryUtility.exponentialRetryAsync(renameDownloadedFileAsyncFn, retries, retryDelayInMs);
+            return RetryUtility.exponentialRetryAsync(renameDownloadedFileAsyncFn, retries * 2, retryDelayInMs);
```

Comment:
If I didn't make a mistake in my calculations, if we make an exponential retry (doubling the delay between each request) with 10 retries and an original delay of 100 ms, in the end the user will wait ~2 minutes.
As of today, all retries included we make the user wait ~3 seconds.
That's a big gap, and I expect that if we were just increasing the total delay to maybe be 12s (7 retries) or even 24 seconds (8 retries), we would already see almost all of the improvement without letting the users wait for nothing if the files are really locked.
From what you've seen, do you expect waiting 2 minutes to make a significant difference compared to waiting 24 seconds to rename the file?

Reply:
Sure, I would just try with 8 retires. I was trying to provide more retires to give ample time to read the folder.